gradio/utils.py

```diff
@@ -593,16 +592,19 @@ def is_update(val):
 
 
 def get_continuous_fn(fn: Callable, every: float) -> Callable:
-    def continuous_fn(*args):
+    # For Wasm-compatibility, we need to use asyncio.sleep() instead of time.sleep(),
```

Comment:
This change is necessary for `get_continuous_fn()` and the `every` param to work in the Wasm env especially in a multi-app situation. However it might affect the core part of this library. If we have to be defensive, there is also an option to keep the original implementation and use this new one only in the Wasm env with `if wasm_utils.IS_WASM:` condition, instead of replacing the entire impl.
WDYT, @abidlabs @aliabd @freddyaboulton ?

Reply:
tested looks good!